test/migration_test.go

```diff
@@ -274,6 +280,49 @@ func TestIntMigration(t *testing.T) {
 		}
 	})
 
+	t.Run("Wait for migration requests to be completed", func(t *testing.T) {
+		// get current time for timeout
+		now := time.Now()
+		migTimeout := 3 * time.Minute
+
+		var requestStatus [10]int
+		requestStatus[0] = len(c)
+		for requestStatus[0] != 0 || requestStatus[1] != 0 || requestStatus[5] != 0 || time.Since(now) > migTimeout {
+			// sleep in order to prevent sqlite database locking
+			time.Sleep(10 * time.Second)
+
+			var migrationStatus []MigrationStatus
+
+			requestStatus = [10]int{0, 0, 0, 0, 0, 0, 0, 0, 0, 0}
+
+			path := fmt.Sprintf("%s/status", base5)
+			req := newreq(t, "GET", srv5, path, nil, nil, headers)
+			resp, err := http.DefaultClient.Do(req)
+			if err != nil {
+				t.Fatalf("HTTP Request Error: %v\n", err)
+			}
+			defer resp.Body.Close()
+
+			err = json.NewDecoder(resp.Body).Decode(&migrationStatus)
+			if err != nil {
+				t.Fatalf("Failed to decode body, %v", err)
+			}
+
+			for _, m := range migrationStatus {
+				requestStatus[m.MigrationStatus] += 1
+			}
+			fmt.Printf("Requests: %d, Successful: %d, Statuses: %v\n", len(migrationStatus), requestStatus[2], requestStatus)
```

Comment:
same argument here, we need explanation for Statuses array `[0 0 3 0 0 0 0 0 0 0]`